config: grant access to specific function in config (#10324)
tarantool/tarantool: changelogs/unreleased/gh-10304-allow-specific-function-access.md

> ## feature/box

Reviewer: Let's use `config` component here and in commit message/pull request.

Author: corrected

> ## feature/box

Author: Docs rewrited

> * Added the ability to provide access to a specific global Lua function
>   in the declarative configuration (gh-10304).

Reviewer 2: ```suggestion
* Now it is possible to grant access to a specific global Lua function
  in the declarative configuration (gh-10304).
```
